### DISCUSSION TITLE
Automate data fetch from airtable, deployed to netlify

### DISCUSSION THREAD

**Discussion by zanewolf** [gatsbyjs/gatsby#35571] [Help]
I have a gatsby website (oceanaffinity.org) that uses airtable as the backend. However, the website only updates new data from airtable (forms submitted by users) when netlify rebuilds the site, because that is when it refreshes the data fetch from airtable. Is there a way to fetch the data from airtable on a regular basis without needing to rebuild the website? 

Or alternatively, a way to trigger a netlify build using gatsby and not airtable? I can trigger a netlify build from airtable using the netlify build hook and scripting, but that requires the pro plan which is a tad expensive. 

Thank you!

**Comment by LekoArts** [comment]
If you want the data to be up-to-date all the time you can use SSR: https://www.gatsbyjs.com/docs/how-to/rendering-options/using-server-side-rendering/

I'd strongly recommend sending a webhook from Airtable on a new entry to trigger a new build -- if this is not available in Netlify I suggest looking at Gatsby Cloud which also offers you build webhooks you can trigger for free: https://www.gatsbyjs.com/products/cloud/
